CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- Merged btsieve into cnd to be able to run comit-rs with a single binary.
```

Comment:
I think the terminology of "merge" was pretty bad from my side :(

How about this:
```suggestion
- Embed btsieve as a library inside cnd: From now on, you'll only need to run cnd to use C( )MIT.
```